compose.yaml

```
@@ -0,0 +1,13 @@
+  mysql:
+    image: 'mysql:latest'
+    environment:
+      - 'MYSQL_DATABASE=wisp_db'
+      - 'MYSQL_ROOT_PASSWORD=123456'
+    ports:
+      - '3306:3306'
+
+  redis:
+    image: 'redis:latest'
+    ports:
+      - '6379:6379'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)